Z/Data/ASCII.hs

```diff
@@ -33,6 +33,11 @@ isASCII :: Word8 -> Bool
 {-# INLINE isASCII #-}
 isASCII w = w <= DEL
 
+-- | @A ~ Z && a ~ z@
+isLatin :: Word8 -> Bool
+{-# INLINE isLatin #-}
+isLatin w = isUpper w && isLower w
```

Comment:
Maybe this funtion should be called `isAlpha` ?

Reply:
`isLatin` is confusing because it may make people believe you are trying to predicate an alpha in Latin encoding(ISO 8859-1), you could add an `isLatinAlpha`, which has a wider value range I believe.

Reply:
_gonna fix to `isAlpha`._ that's my first thought when reading HTTP RFC. i d like it but when i ref to other parsers.